### PR TITLE
show message if slow login detected

### LINKED ISSUE #4976: Introduce a message during login notifying the user of a possible database upgrade
## Abstract

Due to database upgrades when the user logs in the first time with the new version the login might take quite a while.

As a temporary measure, we need a message to appear to the user if the login takes more than 15 seconds. The message could read "A database upgrade might be in progress please wait..." @LefterisJP if you have any other suggestions let us know

## Motivation 

Notify the user that something slow is happening and to not close the app.

## Specification

The message should automatically appear when the login takes more than 15 seconds.
The proper fix will be in the future with #4975

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #4976

To replicate slow login:
put this on the top of `useSessionStore().login()`

```
    if (credentials.username && credentials.password) {
      await new Promise(resolve => setTimeout(resolve, 20000));
    }
```

image

### CODE REVIEW

File: frontend/app/src/components/AccountManagement.vue [hunk 1]
```diff
@@ -23,22 +23,23 @@
           </div>
         </div>
         <connection-loading
-          v-if="!connectionFailure"
+          v-if="!isConnectionFailed"
```
Comment (kelsos) [inline frontend/app/src/components/AccountManagement.vue:26]: can you change this to `hasConnectionFailed` instead

**Review by kelsos — COMMENTED** [review]
some naming request but otherwise it looks ok

File: frontend/app/src/components/AccountManagement.vue (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/components/AccountManagement.vue]: updated

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**